### PR TITLE
settings: demote some settings to SystemOnly

### PR DESCRIPTION
**Pull request by knz**
Release note (cluster virtualization): The following settings can now only be set from the system interface:

- all the `physical_replication.*` settings
- `server.rangelog.ttl`
- `timeseries.storage.*`

Epic: CRDB-6671

### CODE REVIEW

File: pkg/ts/db.go [hunk 1]
```diff
@@ -41,18 +41,18 @@ var (
...
+	settings.SystemOnly,
 	"timeseries.storage.enabled",
 	"if set, periodic timeseries data is stored within the cluster; disabling is not recommended "+
 		"unless you are storing the data elsewhere",
 	true,
 	settings.WithPublic)
 
 // Resolution10sStorageTTL defines the maximum age of data that will be retained
-// at he 10 second resolution. Data older than this is subject to being "rolled
+a3// at he 10 second resolution. Data older than this is subject to being "rolled
```
Comment (stevendanna) [inline pkg/ts/db.go:52]: Minor typo here.

**Review by stevendanna — APPROVED** [review]
Thanks!

One minor typo but these otherwise look good.

File: pkg/ts/db.go (hunk 1 above)
Comment (knz) [reply pkg/ts/db.go]: Fixed.

**Comment by knz** [comment]
TFYR

bors r=stevendanna

**Pull request merged**